android/java/org/chromium/chrome/browser/omnibox/BraveLocationBarQRDialogFragment.java

```diff
@@ -52,6 +52,13 @@ public class BraveLocationBarQRDialogFragment
     // Intent request code to handle updating play services if needed.
     private static final int RC_HANDLE_GMS = 9001;
 
+    // The Android Fragment framework requires a zero-argument constructor to
+    // instantiate fragments. It usually happens on a fragment re-creation
+    // https://github.com/brave/brave-browser/issues/41454
+    public BraveLocationBarQRDialogFragment() {
```

Comment:
If we have any constructor with param, we need to make sure of adding an empty constructor for fragment.